### DISCUSSION TITLE
在非管理员身份使用反向WS情况下改用本地主机而非通配符子域

### DISCUSSION THREAD

**Discussion by MiharaIchiro** [timi137137/AuroraNative#26] [Ideas]
https://github.com/timi137137/AuroraNative/blob/3535c8cc30539e644ca2a529c6e1ca1b8e046f3c/AuroraNative/WebSockets/Server.cs#L53
一般情况下我们机器人、SDK、处理终端都是在一台电脑内，我们可以使用本地主机``"http://localhost:"``以取代使用``通配符子域``防止在非``管理员身份``下引发``AccessDenied``。

**Comment by timi137137** [comment]
这是一个非常好的问题，但是有些情况下127.0.0.1与localhost是有区别的，部分用户可能因为此问题连接不上。故选择使用通配符，也许有什么办法能够解决这个问题？

**Reply by MiharaIchiro** [reply]
建议公开作为参数设置

**Reply by timi137137** [reply]
> 建议公开作为参数设置

是一个好主意，下个版本更新。谢谢

**Comment by timi137137** [comment]
也许你可以试试 #27

**Reply by MiharaIchiro** [reply]
谢谢，在我这里可以正常运作。

**Reply by MiharaIchiro** [reply]
抱歉，后续发现在触发一次``LifeCycle``事件后就没有触发其他事件，原因还在查找中，
具体可能要明天才能再进行滙报。

**Reply by timi137137** [reply]
> 抱歉，后续发现在触发一次LifeCycle事件后就没有触发其他事件，原因还在查找中，
> 具体可能要明天才能再进行滙报。

没事，谢谢你愿意使用这个框架

**Reply by MiharaIchiro** [reply]
经排查后确认是人为问题，是由于``go-cqhttp`` ``post-format`` 改动做 ``array`` 引致。
因为我使用的就是``go-cqhttp``而且是``net462``.NET版本，所以这个框架很符合我的使用条件，而且提供的``API``相对其他``C#``框架足够多，这就是我使用它的原因，也很感谢作者的迅速回覆。
